### ISSUE TITLE
Assignment to constant variable when use none valid void element

### ISSUE THREAD

**Issue by StarpTech** [Prettyhtml/pretty-html-web#6]
```html
<form #heroForm (ngSubmit)="onSubmit(heroForm)">
<label foo="bar" />
  <input type="text" class="input too much classes" [(onChange)]="dede" name="test" />
    <button [style.color]="isSpecial ? 'red' : 'green'"></button></form>
```
will throw in the playground because `label` is used as void element.
```
rpc-wrapper.js:13 Uncaught (in promise) Error: Assignment to constant variable.
    at Parser (prettyhtml.browserified.min.js:1)
    at Function.T.parse (prettyhtml.browserified.min.js:1)
    at prettyhtml.browserified.min.js:1
    at prettyhtml.browserified.min.js:1
    at s (prettyhtml.browserified.min.js:1)
    at Object.run (prettyhtml.browserified.min.js:1)
    at n (prettyhtml.browserified.min.js:1)
    at w (prettyhtml.browserified.min.js:1)
    at Function.T.processSync (prettyhtml.browserified.min.js:1)
    at prettyhtml.browserified.min.js:1
```

**Comment by koca** [comment]
But is this related to playground?

**Comment by koca** [comment]
Cause i'm getting this error in terminal:

```sh

❯ prettyhtml index.html
TypeError: value.indexOf is not a function
    at realLength (/Users/mesut/.config/yarn/global/node_modules/vfile-reporter/index.js:299:22)
    at parse (/Users/mesut/.config/yarn/global/node_modules/vfile-reporter/index.js:157:29)
    at reporter (/Users/mesut/.config/yarn/global/node_modules/vfile-reporter/index.js:66:18)
    at log (/Users/mesut/.config/yarn/global/node_modules/unified-engine/lib/file-set-pipeline/log.js:25:17)
    at wrapped (/Users/mesut/.config/yarn/global/node_modules/trough/wrap.js:25:19)
    at next (/Users/mesut/.config/yarn/global/node_modules/trough/index.js:58:24)
    at FileSet.done (/Users/mesut/.config/yarn/global/node_modules/trough/wrap.js:56:16)
    at FileSet.emit (events.js:182:13)
    at FileSet.one (/Users/mesut/.config/yarn/global/node_modules/unified-engine/lib/file-set.js:118:10)
    at FileSet.emit (events.js:182:13)

```

**Comment by StarpTech** [comment]
@koca I didn't test it yet. I will investigate thanks for validating.

**Comment by StarpTech** [comment]
Fix in 0.2.2
